src/Launcher/launcher_2l.jl

```diff
@@ -89,39 +105,55 @@ function launch_2l!(
         scale = ODE_tol[1] + max(get_abs_max(a_inter), get_abs_max(a)) * ODE_tol[2]
         err   = Δ / scale
 
-        println("Done. Relative integration error err = $(err).")
-        println("Performing sanity checks and measurements ...")
-
-        # terminate if integration becomes unfeasible
-        if err >= 10.0
-            println("Relative integration error has become too large, terminating solver ...")
-            break
-        end
+        println("   Relative integration error err = $(err).")
+        println("   Performing sanity checks and measurements ...")
+        println("   Current vertex maximum Γmax = $(get_abs_max(a_inter)).")
```

Comment:
I would swap the second and third line
```suggestion
        println("   Relative integration error err = $(err).")
        println("   Current vertex maximum Γmax = $(get_abs_max(a_inter)).")
        println("   Performing sanity checks and measurements ...")
```

Reply:
This is makes sense, I think!

Reply:
Finding the vertex maximum is a measurement, so I would vote for sticking to this order

Reply:
Well but it seems odd with the "..." since the statement about the vertex maximum sound somewhat like nothing happens afterwards. I would actually be more convinced by @lgresista 's proposal

Reply:
That was the reason I had the "Performing" line not indented in my original proposal, that would signify, that the vertex amx is a part of the measurements.
Anyways, I do not have a strong opinion on that, so changing is also fine with me.